packages/ui/src/components/SelectInputV2/SearchBarDropdown.tsx

```diff
@@ -78,20 +78,24 @@ export const SearchBarDropdown = ({
       if (!Array.isArray(options)) {
         const filteredOptions = { ...options }
         Object.keys(filteredOptions).map((group: string) => {
-          filteredOptions[group] = filteredOptions[group].filter(option =>
-            option.searchText
-              ? option.searchText.match(regex)
-              : option.value.match(regex),
+          filteredOptions[group] = filteredOptions[group].filter(
+            option =>
+              (option.searchText && !!option.searchText.match(regex)) ||
+              (typeof option.label === 'string' && option.label.match(regex)) ||
+              (typeof option.description === 'string' &&
+                option.description.match(regex)),
```

Comment:
I think it doesn't hurt to search into the value in addition of the rest 😄 

![Screenshot 2024-09-06 at 10 32 13](https://github.com/user-attachments/assets/c5c69ff4-dcf5-4813-b9cc-9c7e49c1a495)

https://github.com/user-attachments/assets/60991418-d3ab-4356-be4f-f531aab78fa7